wallet/types/src/utxo_types.rs

```diff
@@ -31,6 +31,7 @@ pub enum UtxoType {
     ProduceBlockFromStake = 1 << 4,
     CreateDelegationId = 1 << 5,
     DelegateStaking = 1 << 6,
+    TokensOp = 1 << 7,
```

Comment:
I'm not sure what this enum does exactly but UtxoType::TopkensOp looks weird. Should it take spendable outputs from TokensOp instead? It's only IssueNft that can be used from there

Reply:
changed to IssueNft